opensearch/certificates.tf

```diff
@@ -31,7 +40,17 @@ module "certificates" {
     "workers.${local.domain}",
     "${local.resources_namespace}-opensearch",
     "${local.resources_namespace}-opensearch-masters",
-    "${local.resources_namespace}-opensearch-workers"
+    "${local.resources_namespace}-opensearch-workers",
+
+    "audit-opensearch",
+    "audit-opensearch-masters",
+    "audit-opensearch-workers",
+    local.audit_domain,
+    "masters.${local.audit_domain}",
+    "workers.${local.audit_domain}",
+    "${local.resources_namespace}-opensearch-audit",
+    "${local.resources_namespace}-opensearch-audit-masters",
+    "${local.resources_namespace}-opensearch-audit-workers"
```

Comment:
It is a little bit more code, but I would use a completely separate CA and separate certificates for the audit cluster.

This would be more reflective of how it will be in deployed environments